app/content/pages/community.html.erb

```diff
@@ -3,6 +3,7 @@ title: Community
 ---
 
 <%= render Page::ContainerComponent.new(page: current_page) do |page| %>
+  <% page.with_breadcrumbs(page: current_page) %>
```

Comment:
Can't we move the breadcrumb call directly into the `Page::ContainerComponent` so we don't have to add it to every page?

Reply:
I moved it outside, really good idea. I suffered a lot putting in all files the first time : D
I added an options hash to opt out of the breadcrumb for any specific page if needed, in that case it be use like this:

```
<%= render Page::ContainerComponent.new(page: current_page, options: { :render_breadcrumbs => false }) do |page| %>
...
```

Reply:
I guess another approach would to do this and adjust breadcrumbs component accordingly:

```
<%= render Page::ContainerComponent.new(page: current_page) do |page| %>
  <% page.with_breadcrumbs(current_page: nil) %>
```
This would avoid passing the options hash into ContainerComponent, but it seems a bit less readable to me.